bigquery_etl/cli/metadata.py

```diff
@@ -168,8 +190,8 @@ def update(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> None
         if table_metadata_updated:
             table_metadata.write(table_metadata_file)
             click.echo(f"Updated {table_metadata_file}")
-
-    return None
+    except Exception as e:
+        click.echo(f"Error processing {table_metadata_file}: {e}", err=True)
```

Comment:
I think this should raise the exception since that's what's happening now.  Continuing through failure might cause bugs that are hard to track down